vendor/github.com/pivotal/image-relocation/pkg/registry/ggcr/remote.go

```diff
@@ -119,3 +119,13 @@ func resolve(n image.Name) (authn.Authenticator, error) {
 
 	return resolveFunc(repo.Registry)
 }
+
+func getWriteReference(n image.Name) (name.Reference, error) {
+	// if target image reference is both tagged and digested, ignore the digest so the tag is preserved
```

Comment:
I thought that it was the opposite? If the image reference has both `tag` and `digest`, then `tag` is skipped and only the `digest` is used when pulled ? I think that this behavior is due to the fact that a `tag` can be "forced pushed", so the digest may differ ? Maybe am I missing something?

Reply:
You are quite right for the behaviour when _pulling_ an image, but this change relates to the target reference when copying an image, which is analogous to _pushing_ an image. Once we've copied an image in this way, it can be referenced using either the tag or the digest.

Reply:
Ok right, thank you @glyn for the clarification!